### ISSUE TITLE
Add 'no-debug', 'no-skip', 'no-only' rules

### ISSUE THREAD

**Issue by benmonro** [miherlosev/eslint-plugin-testcafe#8]
it seems there are no recent commits on this Repo.  i would be happy to take it over and add some new rules (like no-debug, no-focus-test, no-skip-test etc).  I'm the author & maintainer of eslint-plugin-jest-dom and actively contribute to other eslint plugin repos.  I love testcafe but this plugin needs more rules!

**Comment by benmonro** [comment]
@miherlosev

**Comment by miherlosev** [comment]
Hi @benmonro,

It would be great if you help us improve this plugin. Let's approve a list of rules for adding. I suggest the following rules:

* `no-debug` - restricts the use of the `.debug()` action in tests;
* `no-skip` - restricts the use of the `.skip` hook for tests or fixtures;
* `no-only` - restricts the use of the `.only` hook for tests or fixtures.

Would you please clarify the logic of the `no-focus-test` rule? It's not quite clear what it should do.

**Comment by benmonro** [comment]
Yeah that's the same as no-only.  Jest calls those focused tests and the jest lint rule is called no focused tests.  They are fine for debugging locally but you shouldn't commit them

**Comment by miherlosev** [comment]
We will use the terminology of TestCafe. So, the resulting list of rules is `no-debug`, `no-skip`, `no-only`. These are the most important rules.
The PR from you is greatly appreciated.

**Comment by benmonro** [comment]
Yeah that's fine.
